app/src/main/java/com/asuc/asucmobile/domain/fragments/PlaceholderFragment.java

```diff
@@ -0,0 +1,26 @@
+package com.asuc.asucmobile.domain.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+
+import com.asuc.asucmobile.domain.main.CalendarActivity;
+
+/**
+ * This fragment is used to create a new intent to open the tabbed Calendar Activity because
+ * the Navigation Generator can only open fragments. REEEEEEEE
+ */
+public class PlaceholderFragment extends Fragment {
+    public PlaceholderFragment() {
```

Comment:
can replace with Lombok noargs constructor

Reply:
wow wtf that's cool

Reply:
addressed